packages/react-native/scripts/codegen/generate-artifacts-executor/index.js

```diff
@@ -83,9 +85,14 @@ function execute(
       buildCodegenIfNeeded();
     }
 
-    const libraries = findCodegenEnabledLibraries(pkgJson, projectRoot);
+    const reactNativeConfig = readReactNativeConfig(projectRoot);
+    const codegenEnabledLibraries = findCodegenEnabledLibraries(
```

Comment:
An option would be to let `findExternalLibraries()` in utils.js use `npm react-native config`, which would align the behavior more with the auto linking script and remove a lot of additional code in `findCodegenEnabledLibraries()` but maybe that was a no-go for some reason?

Reply:
the problem is that we do not control the `npm react-native config` command. This is maintained by the CLI. Other frameworks might use different systems for autolinking, so we can't really lock into the `npm react-native config` command.